premis_event_service/views.py

```diff
@@ -597,26 +624,54 @@ def app_agent(request, identifier=None):
                 page = request.GET.get('page')
             else:
                 page = 1
-            atomFeed = makeObjectFeed(
-                paginator=Paginator(Agent.objects.all(), 20),
-                objectToXMLFunction=objectToAgentXML,
-                feedId=requestString[1:],
-                webRoot="http://%s" % request.META.get('HTTP_HOST'),
-                title="Agent Entry Feed",
-                idAttr="agent_identifier",
-                nameAttr="agent_name",
-                request=request,
-                page=page,
-            )
+            try:
+                atomFeed = makeObjectFeed(
+                    paginator=Paginator(Agent.objects.all(), 20),
+                    objectToXMLFunction=objectToAgentXML,
+                    feedId=requestString[1:],
+                    webRoot="http://%s" % request.META.get('HTTP_HOST'),
```

Comment:
I get the desire for consistency, but my tendency is to use single-quotes for anything that looks or behaves like an identifier.

Not arguing it's Right. But that's the rationale.